checkstyle.xml

```diff
@@ -51,7 +51,7 @@
         <!-- Checks for Size Violations.                    -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
         <module name="LineLength">
-            <property name="max" value="100"/>
+            <property name="max" value="120"/>
```

Comment:
👍  I'll switch it back. Seems like we didn't have check-style running on all projects, so I'll need to push some lines around.